### DISCUSSION TITLE
The optimization effect is not good

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#601] [General]
![1](https://github.com/mdolab/dafoam/assets/102606153/0472005b-f294-4a5a-8d2b-1bb51e0bf77a)
![2](https://github.com/mdolab/dafoam/assets/102606153/e56f1073-11a5-46e8-a56b-ca3874b9046b)
I used SST turbulence model flow field inversion, but the optimization effect is not very good, do you know how to improve the optimization process

**Comment by friedenhe** [comment]
We would need more details about your setup (objective, design variables, and constraints, cases, training data, etc).

**Comment by 15175829310** [comment]
[SST-FI.zip](https://github.com/mdolab/dafoam/files/14565787/SST-FI.zip)
Here are my related Settings, please take a look.

**Comment by friedenhe** [comment]
Can you summarize your setup (objective, design variables, and constraints, cases, training data, etc) and post it here? What do you expect to see?

**Reply by 15175829310** [reply]
Thank you for your reply. Here are my settings:
Objective: I want to obtain an inversion field that makes the flow field of the SST model similar to the results of the LES flow field.
The design variable is the beta of the specific dissipation rate generation term in the SST model.
The case is a curved backward-facing step case.

**Comment by friedenhe** [comment]
Are you using the full-flow field for the training data? Also, have you tried to plot the velocity fields for the original SST and the trained SST and see if the trained model improves the velocity field? From the optimization log, it seems that the optimization runs OK because the optimality is dropping, it may just need to run longer.

**Reply by 15175829310** [reply]
I only selected the velocities of 30 points in the flow field as training data.
After training, the speed field has indeed improved.
But its objective function only decreased by about 1%, and I also ran many steps, which I think is abnormal.

**Comment by friedenhe** [comment]
Not sure what caused this.. But it seems that the optimization itself works well, as the optimality is decreasing.. I suggest you show the comparison among the baseline, ref, and optimized velocity profiles. May be that is the best the optimization can do.

**Reply by 15175829310** [reply]
Thank you, I'll take a look again
